tests/contrib/operators/test_gcp_video_intelligence_operator_system_helper.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+import os
+
+import argparse
+from tempfile import NamedTemporaryFile
+
+from tests.contrib.utils.gcp_authenticator import GcpAuthenticator, GCP_AI_KEY
+from tests.contrib.utils.logging_command_executor import LoggingCommandExecutor
+
+GCP_PROJECT_ID = os.environ.get("GCP_PROJECT_ID", "example-project")
+GCP_BUCKET_NAME = os.environ.get("GCP_VIDEO_INTELLIGENCE_BUCKET_NAME", "test-bucket-name")
+GCP_VIDEO_SOURCE_URL = os.environ.get("GCP_VIDEO_INTELLIGENCE_VIDEO_SOURCE_URL", "http://nasa.gov")
+
+
+class GCPVideoIntelligenceHelper(LoggingCommandExecutor):
+    def create_bucket(self):
+        self.execute_cmd(
+            [
+                "gsutil",
+                "mb",
+                "-p",
+                GCP_PROJECT_ID,
+                "-c",
+                "regional",
+                "-l",
+                "europe-north1",
+                "gs://%s/" % GCP_BUCKET_NAME,
+            ]
+        )
+
+        with NamedTemporaryFile(suffix=".mp4") as file:
```

Comment:
You can simplify this a bit without using temporary file
https://stackoverflow.com/questions/18107545/can-i-upload-files-to-google-cloud-storage-from-url. Just would just have to add optional shell=True in execute_cmd. That would be nice.

Reply:
I simplified it using `execute_cmd("bash", "-c", "curl...")`